examples/node/pdf2svg.js

```diff
@@ -70,73 +70,70 @@ ReadableSVGStream.prototype._read = function () {
 };
 
 // Streams the SVG element to the given file path.
-function writeSvgToFile(svgElement, filePath) {
+async function writeSvgToFile(svgElement, filePath) {
   let readableSvgStream = new ReadableSVGStream({
     svgElement,
   });
   const writableStream = fs.createWriteStream(filePath);
-  return new Promise(function (resolve, reject) {
-    readableSvgStream.once("error", reject);
-    writableStream.once("error", reject);
-    writableStream.once("finish", resolve);
-    readableSvgStream.pipe(writableStream);
-  }).catch(function (err) {
+  try {
+    await readableSvgStream.once("error");
+    await writableStream.once("error");
+    await writableStream.once("finish");
+    await readableSvgStream.pipe(writableStream);
+  } catch (err) {
     readableSvgStream = null; // Explicitly null because of v8 bug 6512.
     writableStream.end();
     throw err;
-  });
+  }
 }
 
-// Will be using promises to load document, pages and misc data instead of
+// Will be using async/await to load document, pages and misc data instead of
 // callback.
-const loadingTask = pdfjsLib.getDocument({
-  data,
-  cMapUrl: CMAP_URL,
-  cMapPacked: CMAP_PACKED,
-  fontExtraProperties: true,
-});
-loadingTask.promise
-  .then(function (doc) {
-    const numPages = doc.numPages;
-    console.log("# Document Loaded");
-    console.log("Number of Pages: " + numPages);
+const loadingTask = async function () {
+  const loadTask = pdfjsLib.getDocument({
+    data,
+    cMapUrl: CMAP_URL,
+    cMapPacked: CMAP_PACKED,
+    fontExtraProperties: true,
+  });
```

Comment:
Please just use an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) rather than a named function, i.e.
```javascript
const loadingTask = pdfjsLib.getDocument({
  data,
  cMapUrl: CMAP_URL,
  cMapPacked: CMAP_PACKED,
  fontExtraProperties: true,
});
(async function () {
  try {
    const doc = await loadTask.promise;
    ...
})();
```